ui/shared/Revision.jsx

```diff
@@ -13,7 +13,7 @@ export function AuthorInitials(props) {
   const str = props.author || '';
   const words = str.split(' ');
   const firstLetters = words
-    .map((word) => word.replace(/[^A-Z]/gi, '')[0])
+    .map((word) => word.replace(/\P{General_Category=Letter}/gu, '')[0])
```

Comment:
To keep it working you would need to negate the negate the class.

Reply:
\P makes it negated, no? \p would be a positive class.

Reply:
You are totally right...